x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.test.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import '../../../../__mocks__/kea.mock';
+import { setMockValues } from '../../../../__mocks__';
+import '../../../../__mocks__/enterprise_search_url.mock';
+
+import React from 'react';
+// @ts-expect-error types are not available for this package yet
+import { SearchProvider } from '@elastic/react-search-ui';
+import { shallow } from 'enzyme';
+
+import { SearchExperience } from './search_experience';
+
+describe('SearchExperience', () => {
+  const values = {
+    engine: {
+      name: 'some-engine',
+      apiKey: '1234',
+    },
+  };
+
+  beforeEach(() => {
```

Comment:
v minor q - are we anticipating more tests being added to this file? if not, we may not need a `beforeEach` since there's only 1 test. also not sure if we need the `apiKey` value? (again, ignore me if it's getting added later)

Reply:
I feel like `AppSearchAPIConnector` will fail to instantiate if we don't pass a key, which is why I set `engine.apiKey` IIRC.

We could certainly change this to remove the `beforeEach`, or we could leave it as is, it will make it easier to add tests in the future if/when we need them. I think it's inconsequential either way.

Reply:
We can leave it, no worries!